code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -87,22 +87,20 @@
 		return
 	if((HUSK in target.mutations) || (!ishuman(target)))
 		to_chat(user, "<span class='warning'>Our sting appears ineffective against its DNA.</span>")
-		return 0
+		return FALSE
 	if(ishuman(target))
 		var/mob/living/carbon/human/H = target
-		if(H.species.flags & NO_SCAN) //Prevents transforming slimes and killing them instantly
-			to_chat(user, "<span class='warning'>This won't work on a creature with abnormal genetic material.</span>")
-			return 0
-		if(H.species.flags & NO_BLOOD)
-			to_chat(user, "<span class='warning'>This won't work on a creature without a circulatory system.</span>")
-			return 0
 		if(H.species.flags & NO_DNA)
 			to_chat(user, "<span class='warning'>This won't work on a creature without DNA.</span>")
-			return 0
-	return 1
+			return FALSE
+	return TRUE
 
 /obj/effect/proc_holder/changeling/sting/transformation/sting_action(var/mob/user, var/mob/target)
 	add_logs(user, target, "stung", object="transformation sting", addition=" new identity is [selected_dna.real_name]")
+	var/datum/species/newspecies = all_species[selected_dna.species]
+	if(newspecies.flags & NOTRANSSTING)
+		to_chat(user, "<span class='warning'>The selected DNA is incompatible with our sting.</span>")
+		return FALSE
```

Comment:
This should all be under `can_sting`; you'll also want to ensure there is a `selected_dna` before this can go through (to avoid runtimes)....lastly, be sure to check for NOTRANSSTING *or* the `is_small` var. Keep in mind it was never intended for lings to be able to transform others into monkies.